.stylelintrc.json

```diff
@@ -3,7 +3,6 @@
   "plugins": ["stylelint-prettier"],
   "root": true,
   "rules": {
-    "prettier/prettier": true,
-    "property-no-vendor-prefix": null
```

Comment:
Why remove this?

Reply:
There are no vendor prefixes in the current code, so I think it doesn't make much sense to deactivate this rule.